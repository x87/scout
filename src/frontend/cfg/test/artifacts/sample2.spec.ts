import wrong1 from 'samples/wrong';
import { decompile } from '../util';

`
{$CLEO}

0000:
0002: jump @1
wait 0

:1
end_thread
`;

describe(`wrong 1`, () => {
  it(`loses an unreachable node`, async () => {
    expect(await decompile(wrong1)).toMatchSnapshot();
  });
});
