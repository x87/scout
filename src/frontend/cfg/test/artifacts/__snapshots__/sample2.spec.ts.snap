// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wrong 1 loses an unreachable node 1`] = `
"
:NONAME_0
    NOP
    TERMINATE_THIS_SCRIPT
"
`;
